dotCMS/src/main/java/com/dotcms/rest/api/v1/sites/ruleengine/rules/RuleResource.java

```diff
@@ -82,12 +77,11 @@ protected RuleResource(ApiProvider apiProvider, WebResource webResource) {
     public Map<String, RestRule> list(
             @Context final HttpServletRequest request,
             @Context final HttpServletResponse response,
-            @PathParam("siteId") String siteId) throws DotSecurityException, DotDataException {
+            @PathParam("siteId") String parentId) throws DotSecurityException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "siteId" appears 5 times in this file; the first occurrence is on line 80](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5156545)